src/main/java/io/openliberty/tools/intellij/actions/LibertyGeneralAction.java

```diff
@@ -65,13 +65,16 @@ public void actionPerformed(@NotNull AnActionEvent e) {
                 return;
             }
         }
-        if (buildFile == null) {
+
+        boolean isActionFromShiftShift = "GoToAction".equalsIgnoreCase(e.getPlace());
+
+        if (isActionFromShiftShift || buildFile == null) {
             buildFile = (VirtualFile) e.getDataContext().getData(Constants.LIBERTY_BUILD_FILE);
-            // if still null, prompt for user to select
-            if (buildFile == null) {
+            // if still null, and it is not from shift-shift, then prompt for user to select
+            if (isActionFromShiftShift || buildFile == null) {
                 List<LibertyModule> libertyModules = LibertyModules.getInstance()
                         .getLibertyModules(getSupportedProjectTypes());
-                if (!libertyModules.isEmpty()) {
+                if (isActionFromShiftShift || !libertyModules.isEmpty()) {
```

Comment:
Consider shift+shift and no Liberty modules. A dialog asking the user to select from an empty list? An exception for passing an empty array to the list dialog? I haven't checked what happens because it wasn't possible to enter this block before if the list was empty.

Reply:
Good catch! The change in this if is not really necessary for the fix.